tests/unit/h/views/api/moderation_test.py

```diff
@@ -187,6 +187,24 @@ def test_conflicting_prev_status(
             == "The annotation has been moderated since it was loaded."
         )
 
+    def test_conflicting_prev_status_with_unmoderated_annotation(
+        self,
+        valid_payload_with_prev_status,
+        annotation_context,
+        pyramid_request,
+    ):
+        pyramid_request.json_body = valid_payload_with_prev_status
+
+        with pytest.raises(httpexceptions.HTTPConflict) as excinfo:
+            views.change_annotation_moderation_status(
+                annotation_context, pyramid_request
+            )
+
+        assert (
+            str(excinfo.value)
+            == "The annotation has been moderated since it was loaded."
+        )
```

Comment:
This test is the same as the one above, except that it gets the `annotation_context` fixture, which annotation has `moderation_status = None`, instead of the `moderated_annotation_context one`.

I reckon we could parametrize previous test to pass with both fixtures, but I could not find the right way to parametrize fixtures rather than static values. I'm not sure if that's even possible.